### DISCUSSION TITLE
Baseman not working

### DISCUSSION THREAD

**Discussion by jemharris** [domlysz/BlenderGIS#824] [Q&A]
Hi everyone, 

Im very new to blender and the blenderGIS add-on. I have a background in mapping which is the reason I found the software. 

After downloading and installing the add-on. Ive tried to open the basemap, it only allows me to move it around after pressing escape on my keyboard. Zooming in does not work as the map zooms extremely full and cannot get out of it, the basemap does not load either. Please forgive me if this has been a discussion, I have tried searching but struggling with the technicalities. 

Thank you for any assistance. 

I have uploaded a screen recording of my issue, I am on MacOS Ventura 13.4

https://github.com/domlysz/BlenderGIS/assets/148778459/3258ff04-4688-443a-bf97-2d1693ea4356

**Comment by gebrandtmarkt** [comment]
Hi,
i have exactly the same problem on OS X Sonoma 14.0

Hope there will be a solution soon... Thank you
